docs/can-guides/upgrade/using-codemods.md

```diff
@@ -350,13 +350,13 @@ can-migrate -a **/*.js -t version-4/can-route/page.js
 …will transform the following:
 
 ```js
-can.route(":page", { page: "home" });
+can.route.register(":page", { page: "home" });
```

Comment:
The transform actually handles both since it might run before the `register` transform, but this is fine.

Reply:
That was sloppy on my part, I’ve restored how this guide was before.